static/src/javascripts/boot.js

```diff
@@ -38,7 +39,18 @@ const go = () => {
         const pageViewId: ?string = config.get('ophan.pageViewId');
         const pubData: { browserId?: ?string, pageViewId?: ?string } =
             { browserId, pageViewId };
-        cmp.init({ pubData, isInUsa: isInUsa() });
+
+
+        if (isInAustralia()) {
+            if (config.get('tests.useAusCmpVariant') === 'variant') {
+                cmp.init({ pubData, country: 'AU' });
+            } else {
+                cmp.init({ pubData, country: 'GB' });
```

Comment:
Make it explicit, why we are setting GB in AU.

```suggestion
                cmp.init({ pubData, country: 'GB' }); // Use TCFv2 Framework
```